dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/FuzzyEnumParamConverterProvider.java

```diff
@@ -39,6 +43,10 @@
         final Class<Enum<?>> type = (Class<Enum<?>>) rawType;
         final Enum<?>[] constants = type.getEnumConstants();
         final String parameterName = getParameterNameFromAnnotations(annotations).orElse("Parameter");
+        final Method fromStringM = Stream.of(type.getMethods())
+            .filter(m -> m.getName().equals("fromString") && Arrays.equals(m.getParameterTypes(), new Class[]{String.class}))
+            .findFirst()
+            .orElse(null);
```

Comment:
I was looking for an `Exception`-free equivalent of `type.getMethod("fromString", String.class)`